### ISSUE TITLE
pi bench no longer works. propose its future role.

### ISSUE THREAD

**Issue by codysoyland** [pilosa/tools#67]
The benchmarks are failing:
```
# pi bench basic-query
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x130804b]

goroutine 1 [running]:
github.com/pilosa/tools/vendor/github.com/pilosa/go-pilosa.(*Client).Schema(0xc4201f99f0, 0x1, 0x1, 0xc4201f5260)
	/Users/cody/go/src/github.com/pilosa/tools/vendor/github.com/pilosa/go-pilosa/client.go:275 +0x5b
github.com/pilosa/tools/bench.(*HasClient).Init(0xc420246090, 0xc4201f6b80, 0x0, 0x0, 0x0)
	/Users/cody/go/src/github.com/pilosa/tools/bench/client.go:29 +0xf8
github.com/pilosa/tools/bench.(*BasicQuery).Init(0xc420246090, 0xc4201f6b80, 0x0, 0xc4202435e0, 0x13)
	/Users/cody/go/src/github.com/pilosa/tools/bench/query.go:66 +0x74
github.com/pilosa/tools/bench.RunBenchmark(0x192cb20, 0xc4200240c0, 0xc4201f6b80, 0x0, 0x1926760, 0xc420246090, 0x7)
	/Users/cody/go/src/github.com/pilosa/tools/bench/bench.go:119 +0x177
github.com/pilosa/tools/cmd.NewBasicQueryCommand.func1(0xc420284d80, 0x1d49b68, 0x0, 0x0, 0x0, 0x0)
	/Users/cody/go/src/github.com/pilosa/tools/cmd/basicquery.go:30 +0xf6
github.com/pilosa/tools/vendor/github.com/spf13/cobra.(*Command).execute(0xc420284d80, 0x1d49b68, 0x0, 0x0, 0xc420284d80, 0x1d49b68)
	/Users/cody/go/src/github.com/pilosa/tools/vendor/github.com/spf13/cobra/command.go:698 +0x46d
github.com/pilosa/tools/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420284000, 0x4000100, 0x0, 0xffffffffffffffff)
	/Users/cody/go/src/github.com/pilosa/tools/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/pilosa/tools/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420284000, 0xc42000e010, 0x19245e0)
	/Users/cody/go/src/github.com/pilosa/tools/vendor/github.com/spf13/cobra/command.go:736 +0x2b
main.main()
	/Users/cody/go/src/github.com/pilosa/tools/cmd/pi/main.go:15 +0x74
```

**Comment by travisturner** [comment]
Let's use this ticket to investigate the current state of the `pi` tool and propose how/if we move forward with it.

**Comment by jaffee** [comment]
I think this stuff has been updated with the benchmarking work... can @codysoyland or @alanbernstein close this ticket if that's the case?

**Comment by codysoyland** [comment]
Fixed in #68
